test/jit/test_complex.py

```diff
@@ -55,3 +58,54 @@ def fn(a: int, b: torch.Tensor, dim: int):
         t2 = torch.tensor([0.4, 1.4j, 2.35])
 
         self.checkScript(fn, (t1, t2, 2))
+
+    def test_complex_math_ops(self):
+        vals = ([0.0, 1.0, 2.2, -1.0, -0.0, -2.2, 1, 0, 2]
+                + [10.0 ** i for i in range(2)] + [-(10.0 ** i) for i in range(2)])
+        complex_vals = tuple(complex(x, y) for x, y in product(vals, vals))
```

Comment:
I don't know what's the difference between Python complex literals vs complex class, but `cmath.sqrt(complex(x,y)) and `cmath.sqrt(x+y*1j)` can sometimes return different results:
```
python3 -c "import cmath;x,y=-1.0,-0.0;print(cmath.sqrt(x+y*1j), cmath.sqrt(complex(x,y)))"
1j -1j

```
```suggestion
        complex_vals = tuple(x+y*1k for x, y in product(vals, vals))
```

Reply:
It's because:
```
>>> -0.0 * 1j
(-0+0j)
```
so `x+y*1j` for `x,y=-1.0,-0.0` returns `-1+0j`. Regardless, `cmath.sqrt` returns incorrect value on `-1-0j` and we should probably file a bug for it.